install_files/ansible-base/roles/app/templates/sites-available/focal/source.conf

```diff
@@ -0,0 +1,98 @@
+ServerName {{ securedrop_app_apache_listening_address }}
+<VirtualHost {{ securedrop_app_apache_listening_address }}:80>
+{% if securedrop_app_https_on_source_interface %}
+# Optional HTTPS settings for Source Interface. Requires opt-in
+# via 'securedrop-admin sdconfig', as well as an active certificate.
+# If enabled, all HTTP connections will be redirected to HTTPS.
+RewriteEngine On
+RewriteCond %{HTTPS} off
+RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
+</VirtualHost>
+
+<VirtualHost 127.0.0.1:443>
+# HTTPS config
+SSLEngine on
+SSLCertificateFile /var/lib/ssl/{{ securedrop_app_https_certificate_cert_src|basename }}
+SSLCertificateKeyFile /var/lib/ssl/{{ securedrop_app_https_certificate_key_src|basename }}
+SSLCertificateChainFile /var/lib/ssl/{{ securedrop_app_https_certificate_chain_src|basename }}
+
+# Evaluate support for TLSv1.3 in Tor Browser for Onions, conservatively
+# we'll continue to support TLSv1.2 for now.
+SSLProtocol all -SSLv3 -TLSv1 -TLSv1.1
```

Comment:
TLSv1.3 support deserves a spike, but I don't see it as a release-blocker. Probably worth its own issue, though. cc @emkll

Reply:
Agreed, a good topic for follow-up. But right now, we only support TLS 1.2 which remains safe. TLS1.3 only would vastly improve cipher suite configuration however, see below.